src/Mod/Draft/DraftEdit.py

```diff
@@ -1209,62 +1232,97 @@ def updateDimension(self,v):
 
     # SKETCH: just if it's composed by a single segment-------------------------
 
-    def setSketchPts(self, obj):
+    def getSketchPts(self, obj):
+        '''
+        returns the list of edipoints for the given single line sketch (WallTrace)
+        0 : startpoint
+        1 : endpoint
+        '''
+        editpoints = []
         if obj.GeometryCount == 1:
-            self.editpoints.append(self.pl.multVec(obj.getPoint(0,1)))
-            self.editpoints.append(self.pl.multVec(obj.getPoint(0,2)))
+            editpoints.append(obj.getGlobalPlacement().multVec(obj.getPoint(0,1)))
+            editpoints.append(obj.getGlobalPlacement().multVec(obj.getPoint(0,2)))
+            return editpoints
         else:
             FreeCAD.Console.PrintWarning(translate("draft","Sketch is too complex\
-                 to edit: it is suggested to use sketcher editor")+"\n")
+                 to edit: it is suggested to use sketcher default editor")+"\n")
```

Comment:
Continuation inside the parentheses can be done with strings as well, without the backslash.

```py
FreeCAD.Console.PrintWarning(translate("draft",
                                       "Sketch is too complex"
                                       "to edit: it is suggested to use sketcher default editor")+"\n")
```